contents/copy/scholarships/guide.md

```diff
@@ -1,3 +1,114 @@
+- Thursday to Monday for combo ticket holders
+- Thursday to Saturday for CSSconf ticket holders
+- Friday to Monday for JSConf ticket holders
+
+(that is, unless you have private accommodation). In case you need assistance with regards to the hotel, you can contact Simone (simone@jsconf.eu).
+
+If you're arriving by public transport, the closest stations are:
+
+- **Berlin Ostkreuz** (S-Bahn, Bus) — One of the biggest train stations in Berlin, it's serviced by the S3, S5, S7, Ring (S41, S42), and more S-Bahn lines.
+- **Neue Bahnhofsstr.** (Tram, Bus) — Situated right next to the hotel and serviced by the 21 tram and the 240 bus line.
+
+You will only need an *AB Ticket* to get there. For more information, refer to the transport section.
+
+## Venue Information
+
+### Arena Berlin
+
+*Eichenstraße 4, 12435 Berlin*
+
+The entire event, including meals, parties, and everything else takes place in here, so there's no long distances you have to take during the day.
+
+If you're travelling to the venue by taxi, most taxi drivers should recognize the destination if you just say "Arena Berlin". The closest public transport stations are:
+
+- **Treptower Park** (S-Bahn, Bus) — Serviced by the S8, S9, and the Ring lines (S41 and S42). From there, you can walk or take the 165/265 bus lines.
+- **Eichenstraße/Puschkinallee** (Bus) — Serviced by exactly those 165/265 lines, as well as the 104 line. From here, the venue is a stone throw away. This station is also reachable from Schlesisches Tor.
+
+## Bookings & Reimbursements
+
+If you agreed to it, we have booked the hotel for you. In case you're unable to affort transportation costs, such as taxis or public transport, please contact Simone and we'll figure something out together!
+
+## Contacts
+
+If you have any questions or need help, feel free to talk to us! We're all on the "wwwtf.berlin" Slack instance that you've been invited to.
+
+- **Simone Haas,** simone@jsconf.eu, `@haas_simone` on Twitter
+- **Olivia Hugger**, `@serbal_chan` on Twitter
+- 
+
+## Parties and Social Events
+
+**Thursday, May 31: CSSconf EU Warm-Up**
+
+On Thursday evening you are invited to join us for our pre-conf get together, a great opportunity to meet the team, each other, and fellow attendees prior to the conference. _More details coming soon!_
+
+**Friday, June 1: CSSconf EU Garden Dinner, 6:40pm / JSconf EU Opening Evening, 9pm**
+
+What would a great day be if it didn’t end in celebration! At the end of the day we'll have dinner in our outdoor area at the river, followed by the JSconf Opening, which is of course open to all CSSconf attendees!
+
+**Saturday, June 2: JSConf EU Main Party, 9pm**
+
+We invite you to celebrate with us and the community at the shared JSConf + CSSconf party.
+
+**Sunday, May 7: JSConf EU Closing Party**
+
+We’ll close the long conference weekend with a final party! _More details coming soon!_
+
+**Monday, May 8: relax.js Brunch, 10.30am-2pm, Datscha Xberg**
```

Comment:
Is this even relevant for scholarship holders?

Reply:
The whole schedule is relevant for scholarship holders. But the event is not scholarship related. Perhaps it would be better to point to the planned scholarship meet & greet at the community lounge before the evening program of JSConf EU starts on Saturday. All scholarship holders (even those who just have a ticket for CSSconf) can take part. We have no final schedule for that yet